README.md

```diff
@@ -2,22 +2,406 @@
 
 [![CircleCI](https://circleci.com/gh/DataDog/dd-trace-rb/tree/master.svg?style=svg&circle-token=b0bd5ef866ec7f7b018f48731bb495f2d1372cc1)](https://circleci.com/gh/DataDog/dd-trace-rb/tree/master)
 
-``ddtrace`` is Datadog’s tracing client for Ruby. It is used to trace requests as they flow across web servers,
-databases and microservices so that developers have great visiblity into bottlenecks and troublesome requests.
+`ddtrace` is Datadog’s APM tracing client for Ruby. It is used to trace requests as they flow across web servers, databases and microservices so that developers have great visiblity into bottlenecks and troublesome requests.
+
+## How it works
+
+For a basic overview of what Datadog APM is and how it works, check out our [APM documentation][apm docs].
+
+`ddtrace` wraps Ruby code to take measurements of individual operations within your Ruby application. These measurements, called *spans*, are joined together to form a *trace*, which is sent to the Datadog agent.
+
+Here's an example of a manually traced web request:
```

Comment:
Any specific reason we start with Manually Traced Web Request?
I'd suggest replacing it with a config file example for Rails or any other common integration, as for a first time user, it will be easy to grasp.

Reply:
The idea I had here for "How it Works" isn't so much "how to quickly use it" but "what does it do and what purpose does it serve?" Ideal for the user who doesn't know what the purpose of the library is, which is usually the first question I think most engineers as when looking at a project for the first time.

I think we could add one more snippet to this section that lets them know "there's also some auto-instrumentation, too, and its this easy" but doing so might risk cluttering what was meant to be a TL;DR into something a user might say "TL;DR".